container_api/src/container.rs

```diff
@@ -302,10 +296,10 @@ impl Logger for NullLogger {
 }
 
 fn create_memory_context(
-    _: &String,
+    agent_name: &String,
     network_config: JsonString,
 ) -> Result<Context, HolochainError> {
-    let agent = AgentId::generate_fake("c+bob");
+    let agent = AgentId::generate_fake(agent_name);
```

Comment:
Hooking up this param actually made all the difference when one of the JS tests was hanging. Does that make sense?

Reply:
So, wait... we were passing in agent ids, but it wasn't using them , it was just hardcoded in here?

Reply:
I've replaced this function altogether by the `ContextBuilder` in https://github.com/holochain/holochain-rust/pull/761. Just FYI.